src/rule-must-colocate-fragment-spreads.js

```diff
@@ -153,15 +167,33 @@ function rule(context) {
 
     ImportDeclaration(node) {
       if (node.importKind === 'value') {
-        foundImportedModules.push(utils.getModuleName(node.source.value));
+        node.specifiers.forEach(specifier => {
+          if (
+            allowNamedImports &&
+            specifier.imported &&
+            isFirstLetterUppercase(specifier.imported.name)
```

Comment:
The first letter of an import that references a fragment does not always start with an upper case letter.

For example, you can define a hook `useXyz(ref: xyzFragment$key)`.